usd/src/usd_parser/USDLight_TEST.cc

```diff
@@ -70,7 +70,7 @@ TEST(USDLightsTest, DistanceLight)
   EXPECT_EQ(ignition::math::Pose3d(0, 0, 10, 0, IGN_DTOR(45), 0),
       light->RawPose());
 
-  prim = stage->GetPrimAtPath(pxr::SdfPath("/shapes/diskLight"));
+  prim = stage->GetPrimAtPath(pxr::SdfPath("/diskLight"));
```

Comment:
Since we no longer have the `shapes` top level prim in the testing file, does the `schemaToStop` in the corresponding `ParseUSDLights` call need to be updated? It's currently `/shapes`, which doesn't exist: https://github.com/ignitionrobotics/sdformat/blob/a8249fbc63ce23b01a73df2344f0bcc63a4c9fab/usd/src/usd_parser/USDLight_TEST.cc#L76-L77